snappiershot/serializers/constants.py

```diff
@@ -1,10 +1,11 @@
 """ Constant values used by the serializers. """
 import datetime
 from abc import ABC
-from typing import Any, Dict, Iterator, List, NamedTuple, Set, Union
+from decimal import Decimal
+from typing import Any, Dict, Iterator, List, NamedTuple, Set, Tuple, Union
 
 _Primitive = Union[bool, float, int, None, str]
-JsonType = Union[_Primitive, Dict[str, Any], List[Any]]
+JsonType = Union[_Primitive, Dict[str, Any], List[Any], Tuple[Any, ...]]
```

Comment:
I don't think changing this type signature affecting anything, so I'd revert it

Reply:
Forgot I did this! Good catch.